doc/install.text

```diff
@@ -0,0 +1,197 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+The main goal of this quick-start guide is to enable the user to extract and 
+wavelength calibrate an Arc spectrum.
+
+* Install the LSST binary distribution >= 12.0 (https://pipelines.lsst.io/install/conda.html)
+  Here it should not matter whether you install anaconda or miniconda. Please note that
+  there's only a need to install lsst-distrib, not lsst-sims.
+  Don’t forget to setup the LSST environment as mentioned on the website. You will need
+  to do this every time you want to use the pipeline in a new shell, so you might want
+  to add these 2 lines to your .bashrc:
+
+source activate lsst
+source eups-setups.sh
+
+
+* The test data are quite large (~38 MB) and are stored in git using git-lfs. We therefore
+  need to install git-lfs (https://git-lfs.github.com/). Please follow the instructions at
+  https://git-lfs.github.com to installing git-lfs (e.g. using homebrew on os/x), then type
+  
+  git lfs install
+
+  There's no need to issue any "git lfs track" commands
+
+
+* On Linux, all the anaconda packages were compiled with gcc 4.small and are incompatible
+  with modern versions. Therefore, if you are installing the PFS DRP on a Linux machine, 
+  install gcc in anaconda:
+
+conda install gcc
+
+
+* For convenience of this quick-start guide, create a directory for the PFS-DRP repositories:
+
+export PFS_DRP=<your_target_directory>
+(e.g. export PFS_DRP=$HOME/PFS
+mkdir -p $PFS_DRP
+cd $PFS_DRP
+
+
+* Clone the relevant git repositories. Note that currently a newer version of sconsUtils
+  is needed to suppress some irrelevant compiler warnings. Installing and setting up
+  sconsUtils will be obsolete in the next binary LSST Stack installation.
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+git clone git@github.com:lsst/sconsUtils.git
+
+* The next 2 steps are temporary workarounds until fixed:
+* Set $EUPS_PKGROOT
```

Comment:
(Ok, the answer is because Robert told you to, but I think basically the same logic applies: all of this should become obsolete with the next binary release.)